### ISSUE TITLE
Inconsistent sizes of screenshots with each other of chrome and firefox

### ISSUE THREAD

**Issue by sagarvijaygupta** [marco-c/autowebcompat#149]
For images which have different sizes:
https://gitlab.com/marco-c/autowebcompat-data/blob/master/1022_firefox.png
https://gitlab.com/marco-c/autowebcompat-data/blob/master/1022_chrome.png

> File "label.py", line 50, in get_new_image
    img_blended = Image.blend(background, overlay, 0.5)
  File "/home/sagar/anaconda3/envs/autowebcompat/lib/python3.6/site-packages/PIL/Image.py", line 2390, in blend
    return im1._new(core.blend(im1.im, im2.im, alpha))
ValueError: images do not match

**Comment by Shashi456** [comment]
@marco-c  do you think inconsistent sizes overall are because of how the crawler interacts with the browser?

**Comment by marco-c** [comment]
I'm not sure what the exact cause is.

**Comment by sagarvijaygupta** [comment]
There are 143 such types of inconsistencies. Should we just remove them? And put a check in `collect.py` so that next time no such screenshots are stored until we get the exact reason for this problem.

**Comment by marco-c** [comment]
No, I'd prefer to keep them. Some of them might be valid after cropping (#115).
